samples/charts/data-chart/scatter-bubble-chart-single-source/App.razor

```diff
@@ -0,0 +1,74 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="igContainer">
+    <div class="igOptions-vertical">
+        <span class="igLegend-title">Annual working hours vs. GDP per capita, 2015</span>
+    </div>
+    <div class="igComponent">
+
+        <DataChart Height="100%" Width="100%"
+                   Legend="LegendRef"
+                   IsHorizontalZoomEnabled="true"
+                   IsVerticalZoomEnabled="true">
+
+            <NumericXAxis Name="xAxis" Title="GDP" MaximumValue="100000" Interval="50000" AbbreviateLargeNumbers="true" />
+            <NumericYAxis Name="yAxis" Title="Total Population" AbbreviateLargeNumbers="true" MinimumValue="1400.00" MaximumValue="2400.00" Interval="200" />
+
+            <BubbleSeries XAxisName="xAxis"
+                          YAxisName="yAxis"
+                          DataSource="@DataSource"
+                          XMemberPath="GDPPerCapita"
+                          YMemberPath="AverageHoursWorked"
+                          MarkerType="MarkerType.Circle"
+                          MarkerOutline="rgb(186,231,114)"
+                          RadiusScale="SizeScale"
+                          RadiusMemberPath="TotalPopulation"
+                          FillMemberPath="TotalPopulation"
+                          FillScale="@FillScales[0]">
```

Comment:
use named fields when binding instead of List in your samples:
` FillScale="@EuropeSizeScale" `